installer/windows/scripts/configure_acrylic.bat

```diff
@@ -18,7 +25,7 @@ if not exist "%acrylic_config%" (
 FINDSTR /c:".massa" "%acrylic_config%" >nul 2>&1
 if %errorlevel%==0 (
     ECHO TLD already in the file
-    goto :EOF
+    EXIT 0
```

Comment:
@thomas-senechal I change this to exit 0, do you think it's a problem?

Reply:
Looks good to me !